tests/logictest/requirements.txt

```diff
@@ -1,9 +1,9 @@
-# TODO: specify version of dependencies
-mysql-connector
-six
-PyHamcrest
-requests
-environs
+# This file need to be manually keeped update with pyproject.toml
```

Comment:
In the ci logical test the docker use this docker file: tests/logictest/Dockerfile https://github.com/datafuselabs/databend/blob/main/tests/logictest/Dockerfile

If only modifying the requirements.txt, It maybe not be valid.

Reply:
ok, the dockerfile should use `requirements.txt` instead.